src/datapump/jobs/version_update.py

```diff
@@ -31,7 +33,8 @@ class RasterVersionUpdateJob(Job):
     content_date_range: ContentDateRange
     tile_set_parameters: RasterTileSetParameters
     tile_cache_parameters: Optional[RasterTileCacheParameters] = None
-    aux_tile_set_parameters: List[RasterTileSetParameters] = []
+    aggregated_tile_set_parameters: AuxTileSetParameters = None
```

Comment:
For correct typing, change:

AuxTileSetParameters -> Optional[AuxTileSetParameters]